server/src/termdb.filter.js

```diff
@@ -128,7 +128,7 @@ function get_categorical(tvs, CTEname, ds, onlyChildren) {
 	FROM anno_categorical 
 	WHERE term_id = ?
 	AND value ${tvs.isnot ? 'NOT' : ''} IN (${tvs.values.map(i => '?').join(', ')})`
-	if (onlyChildren && ds.cohort.db.tables.has('sample_ancestry')) query = getChildren(query)
+	if (onlyChildren && ds.cohort.termdb.hasAncenstry) query = getChildren(query)
```

Comment:
where is `ds.cohort.termdb.hasAncenstry` property set? also it should be called `hasSampleAncestry` to be distinguished from existing "ancestry" table

Reply:
I renamed the property now, it is initialized in termdb.server.init.js
`		ds.cohort.termdb.hasSampleAncestry = ds.cohort.termdb.sampleTypes.size > 1
`